R/sanitize_display.R

```diff
@@ -107,16 +107,16 @@ expand_subsets_agm_df <- function(
     dplyr::select(-present) %>%
     ungroup() %>%
     # Back-fill the subset_type field to match what's in the subset_value field
-    # when subset_value is "All Students." This is pretty hacky, but necessary
-    # because "All Students" does not appear in the subset_config, and thus
+    # when subset_value is "All." This is pretty hacky, but necessary
+    # because "All" does not appear in the subset_config, and thus
     # there's nowhere else to take this value from. We also know the subset_type
-    # for "All Student" will always be "All Students." When we change this
+    # for "All Student" will always be "All." When we change this
     # string for Catalyze, the principle will be the same — whatever string we
```

Comment:
nit: update this comment about "when we change this string for Catalyze" b/c it's been changed now.

Reply:
3671042